src/components/UI/LoadingSpinner/LoadingSpinner.js

```diff
@@ -0,0 +1,11 @@
+import './LoadingSpinner.scss';
+
+const LoadingSpinner = () => {
```

Comment:
Good catch adding a loading spinner for better UX!

One accessibility nit: there would be no info for screenreader users here. How could we make this more accessible?